### DISCUSSION TITLE
Social Media Info of Networks

### DISCUSSION THREAD

**Discussion by koltigin** [ping-pub/explorer#518] [General]
It is available on the social media information pages of some networks. However, social media information of another network appears on another network. Can we correct the parameters of this information? If there is such an opportunity, where will we do it?

**Comment by liangping** [comment]
All social media information are fetched from coingecko.com, You can correct information there if you want.

**Reply by koltigin** [reply]
I entered the id of the projects that have pages on Coingecko, but sometimes it doesn't work. I think it's because of coingecko.
